### PR TITLE
[WNMGDS-919] Medicare focus styles

### PR DESCRIPTION
**Pull request by line47**
Medicare will be inheriting the focus styles from the core design system when they are ready. 

We are waiting on review and should know by July 12, 2021

- Removed `:focus` styles that are no longer needed because they were duplicative of the SCSS set up in core 

### How to test 
- Run the M.gov doc site locally and see that the eggplant color is used

### CODE REVIEW

File: src/styles/base/_override.links.scss [hunk 1]
```diff
@@ -1,49 +1,21 @@
-%link-inverse {
```
Comment (line47) [inline src/styles/base/_override.links.scss:1]: Removed duplicative code from the links override while i was in there for focus styles anyways

File: src/styles/base/_override.links.scss [hunk 2]
```diff
@@ -1,49 +1,21 @@
...
-    border-color: $focus-choice-color;
-    outline: none;
   }
 
   &.mct-link--inverse {
     @extend %link-inverse;
   }
 }
 
-a,
```
Comment (line47) [inline src/styles/base/_override.links.scss:41]: Removed the explicit A style and set `$ds-include-base-html-rulesets: true;` in the SCSS build file. That is the preferred way to add A styling.

File: src/styles/index.scss [hunk 3]
```diff
@@ -8,8 +8,9 @@
  */
 
 // Settings: Globally-available variables, mixins, functions and config (Sass code that doesn't output CSS)
-@import "settings/index";
+
 @import "@cmsgov/design-system/dist/scss/settings/index";
+@import "settings/index";
```
Comment (line47) [inline src/styles/index.scss:13]: needed to set the mgov theme styles after setting up the design sytem

File: src/styles/settings/_variables.build.scss [hunk 4]
```diff
@@ -1,2 +1,2 @@
 // Includes rulesets to base html (i.e. applies styles to <a>)
-$ds-include-base-html-rulesets: false;
+$ds-include-base-html-rulesets: true;
```
Comment (line47) [inline src/styles/settings/_variables.build.scss:2]: Setting this to true to include the A styling

File: src/styles/settings/_variables.color.scss [hunk 5]
```diff
@@ -96,7 +94,7 @@ $color-error-lightest: $red-100;
 $color-visited: $color-primary-alt-light;
 
 $color-base: $color-gray-dark;
-// Due to missing !default in core, this wont work until core fixes it
```
Comment (line47) [inline src/styles/settings/_variables.color.scss:99]: We've fixed this !default flag in core

File: src/styles/settings/_variables.color.scss [hunk 6]
```diff
@@ -13,11 +13,9 @@ $color-primary-darkest: darken($color-primary, 20%);
 $color-primary-light: $teal-300;
 $color-primary-lightest: $teal-100;
 
-// Focus
-$focus-choice-color: #f1ad15;
-$focus-choice-color-highlight: rgba(241, 173, 21, 0.4);
-$focus-text-field-color: $color-primary;
-$focus-text-field-color-highlight: rgba(64, 158, 144, 0.4);
+// Focus colors
```
Comment (line47) [inline src/styles/settings/_variables.color.scss:16]: Set custom focus colors

**Comment by davidakennedy** [comment]
I checked out the Global Header component and I don't see any custom focus styles, beyond the ones already commented out, which we'll remove as this PR progresses.

**Review by nichia — COMMENTED** [review]
The focus styles looks great. Some issues I've noted below:
- backwards-compatibility, maybe keep the existing variables for now?
- global header focus not showing when the action button is clicked and which displays the `alert` box callback
Screen Shot 2021-06-24 at 8 46 25 AM

File: src/styles/settings/_variables.color.scss [hunk 7]
```diff
@@ -13,11 +13,10 @@ $color-primary-darkest: darken($color-primary, 20%);
...
```
[the remark this answers is not in the document]
Comment (line47) [reply src/styles/settings/_variables.color.scss:16]: I just ran through this again. 
- Built the core design system locally 
- linked Mgov to the local version of core 
- This is what the global header looked like for me (focus styles working) 
Screen Shot 2021-06-25 at 1 29 15 PM

**Comment by line47** [comment]
Melissa said the eggplant color was [approved for Mgov design system in CMS Slack today ](https://cmsgov.slack.com/archives/CT169N26L/p1628525136019900)

**Review by zarahzachz — APPROVED** [review]

**Pull request closed without merging**